wordcount/count/BenchmarkValues.cs

```diff
@@ -0,0 +1,44 @@
+using System.Buffers;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace BenchmarkData;
+
+public static class BenchmarkValues
+{
+    public static int Size => 16 * 1024;
+
+    public static SearchValues<byte> WhitespaceSearchAscii = SearchValues.Create((ReadOnlySpan<byte>)[9, 10, 11, 12, 13, 32, 194, 225, 226, 227]);
```

Comment:
Values 194, 225, 226, and 227 should not be present. (Might want to leave a comment as to where you got these values from!)

Reply:
I'll just delete that. There are no references to it.